integration-tests/rpc-nodes.spec.ts

```diff
@@ -16,16 +17,33 @@ CONFIGS().forEach(
     knownBigMapContract,
     knownSaplingContract,
     knownViewContract,
-    txRollupAddress,
   }) => {
     const Tezos = lib;
 
     const Limanet = protocol === Protocols.PtLimaPtL ? it : it.skip;
-
+    const MumbaiAndAlpha = protocol === Protocols.PtMumbaii || protocol === Protocols.ProtoALpha ? test : test.skip;
```

Comment:
nitpick, but M should be lowercase since it's a function and not a class/type.

Reply:
also this file line 443 and onward is the integration test :)